src/transformStateWithClones.js

```diff
@@ -7,7 +7,44 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  // Here I clone the given state / создаю объект текущего состояния
+  let currentState = { ...state };
```

Comment:
Try not to use let in this case (Mean search is another way to transform the currentState object).